### ISSUE TITLE
Support underscores on column names in Google Spreadsheets

### ISSUE THREAD

**Issue by ggalmazor** [opendatakit/aggregate#443]
As per [conversation on the forums](https://forum.opendatakit.org/t/google-spreadsheet-removes-underscores-from-field-names/18345/2), it looks like we're replacing allegedly unsafe characters, including: `*`, `:`, `-`, and `_` in column names when publishing to Google Spreadsheets.

We should try to understand why this is being currently done and whether we need to continue doing it or not.

**Comment by yanokwa** [comment]
@ggalmazor Where in the code is this done? Maybe `git blame` would help figure this out. If not, I'm hoping this is an easy thing for you to test.

@wbrunette Do you know off the top of your head?

**Comment by ggalmazor** [comment]
I see a Dylan Price on the git blame. Does that ring a bell?

**Comment by yanokwa** [comment]
Yeah, I know Dylan, but I think it's too much work to track him down. We should just try underscores and special characters and update the code accordingly.

**Comment by ggalmazor** [comment]
OK, I'm tagging this issue for v2.1.0 so that we don't forget to give it a go before dismissing or fixing it.

**Comment by wbrunette** [comment]
Apologies for the delayed response. Old history from memory, not sure what is in the code now and haven't looked. 

Column headings were originally the variable names in the xform (similar to how database columns operate). A feature request that got added was to change to the question vs the variable names in the spreadsheet. 

Back to dashes and underscores ... 8-9 years ago Google spreadsheets did not support either dashes or underscores (can't remember which way it was) being in the column name and to make things complicated Google fusion tables did not accept the other, so we converted the dashes to underscores and underscores to dashes depending on which service we were exporting to.
